### ISSUE TITLE
draw filled polygons for airport areas

### ISSUE THREAD

**Issue by almccon** [stamen/terrain-classic#30]
Where available from OSM. Will cover over landcover issues in airports:

![screen shot 2015-09-18 at 6 03 46 pm](https://cloud.githubusercontent.com/assets/1212178/9973861/ba5e27be-5e2f-11e5-84be-fb63140792a8.png)

**Comment by clhenrick** [comment]
@almccon do you happen to know which tag OSM uses for airport fills?

**Comment by almccon** [comment]
Try `aeroway=aerodrome`, I think. http://www.openstreetmap.org/way/131190417

**Comment by clhenrick** [comment]
@almccon it looks like airport fills are linestring geometries so we can't apply a polygon-fill with CartoCSS (or is there a hack where we can?)

I suppose an alternative would be to create a separate table that converts these features to polygon geometries so we can render the fills? 

```
terrain_seattle=# select distinct st_geometrytype(geometry) from osm_aeroways where type = 'aerodrome';
 st_geometrytype 
-----------------
 ST_LineString
(1 row)
```

**Comment by clhenrick** [comment]
referenced the wrong issue for this commit e13c8d9 but I added temporary cartoCSS to style the airport fills.

**Comment by clhenrick** [comment]
Actually I'm wrong, the fill is working for airport areas / aerodromes.

**Comment by clhenrick** [comment]
@almccon in looking into this I noticed:  
- that airport runways were covering up some minor roads, rail, and tunnels at higher zooms so I moved the runway layer below the roads layer in the respective `.yml` files. 
- the layer containing runways was applying default line styles to the area boundaries which I removed.
- I added a second layer and polygon style for the airport areas (the runway color lightened by 10%) with no line style.

Example:
![screen shot 2015-10-26 at 1 28 52 pm](https://cloud.githubusercontent.com/assets/161748/10736878/a20fa77c-7be5-11e5-8acb-3ac31dcc5813.png)
![screen shot 2015-10-26 at 1 29 06 pm](https://cloud.githubusercontent.com/assets/161748/10736879/a45feb72-7be5-11e5-8b4c-c132546688ee.png)
